src/display/api.js

```diff
@@ -1942,6 +1942,7 @@ class WorkerTransport {
     this.loadingTask = loadingTask;
     this.commonObjs = new PDFObjects();
     this.fontLoader = new FontLoader({
+      ownerDocumentContext: params.ownerDocumentContext,
```

Comment:
Please move this to the bottom of the list instead, to be consistent with ordering used in ` src/display/font_loader.js`.

Reply:
Will do